internal/metrics/metricsManager_test.go

```diff
@@ -68,9 +67,9 @@ var _ = DescribeTable(
 		// Verify that the label was added:
 		metrics := server.Metrics()
 		Expect(metrics).To(MatchLine(
-			`^service_assisted_installer_host_installation_phase_seconds_count\{.*discoveryAgentVersion="v1.0.0-141".*\} .*$`,
+			expectedMetricRegex,
 		))
 	},
-	Entry("Full image reference", "quay.io/edge-infrastructure/assisted-installer:v1.0.0-141"),
-	Entry("Tag only", "v1.0.0-141"),
+	Entry("Starting installation phase", models.HostStageStartingInstallation, `^service_assisted_installer_host_installation_phase_seconds_count\{.*phase="Starting installation".*\} .*$`),
```

Comment:
Not really relevant, but I believe that very long lines are hard to read and change. I'd suggest to put each argument in a separate line:

```go
Entry(
        "Starting installation phase",
        models.HostStageStartingInstallation,
        `...`,
),
```